sklearn/random_projection.py

```diff
@@ -82,7 +82,7 @@ def johnson_lindenstrauss_min_dim(n_samples, *, eps=0.1):
         Number of samples. If an array is given, it will compute
         a safe number of components array-wise.
 
-    eps : float or numpy array of float in ]0,1[, optional (default=0.1)
+    eps : float or numpy array of float in (0,1), default=0.1
```

Comment:
```suggestion
    eps : float or ndarray of shape (n_components,), dtype=float, default=0.1
```
Could you put the information regarding the range in the docstring which follows.